sync-core/src/main/java/com/cloudant/sync/query/UnindexedMatcher.java

```diff
@@ -258,7 +265,7 @@ private boolean executeSelectorTree(QueryNode node, DocumentRevision rev) {
     private boolean valueCompare(Object actual, String operator, Object expected) {
         boolean passed;
 
-        if (operator.equals("$eq")) {
+        if (operator.equals("$eq") || operator.equals("$in")) {
```

Comment:
Replaced all literals with constants in the `if-else if...` block.  Flipped `$in` to `$eq` in the calling routine.  Done in 5b5040e6a64ad6720ef17ace362f2020397d0880.